scripts/create-darwin-volume.sh

```diff
@@ -92,8 +128,9 @@ main() {
         echo "     ------------------------------------------------------------------ "
         echo ""
         echo "  1. Remove the entry from fstab using 'sudo vifs'"
-        echo "  2. Destroy the data volume using 'diskutil apfs deleteVolume'"
-        echo "  3. Remove the 'nix' line from /etc/synthetic.conf or the file"
+        echo "  2. Remove /Library/LaunchDaemons/org.nixos.darwin-store.plist"
+        echo "  3. Destroy the data volume using 'diskutil apfs deleteVolume'"
+        echo "  4. Remove the 'nix' line from /etc/synthetic.conf (or the file)"
```

Comment:
```suggestion
        echo "  2. Run `sudo launchctl bootout system/org.nixos.darwin-store`
        echo "  3. Remove /Library/LaunchDaemons/org.nixos.darwin-store.plist"
        echo "  4. Destroy the data volume using 'diskutil apfs deleteVolume'"
        echo "  5. Remove the 'nix' line from /etc/synthetic.conf (or the file)"
```

Reply:
Removing the KeepAlive setting nullifies this though, yeah?

Reply:
It makes not doing this less harmful, but it's still a good idea to unload the job before deleting the plist. Especially if the user is going to reinstall before rebooting, I think if the job is still loaded `launchctl` will just error on the `bootstrap`.

Reply:
Good point on reinstalling. I was just thinking it made sense to save them a step, but it'd definitely undercut people just trying to get the system clean to reinstall.